packages/core/stories/button.doc.stories.mdx

```diff
@@ -32,61 +26,29 @@ A Button executes an action when clicked. Button labels should use only one or t
 Button contains three variants `cta`, `primary` and `secondary`. If not specified, the default value is `primary`.
 
 <Canvas>
```

Comment:
Problem with this approach is that if user view the code , they don't see anything helpful, just a bunch of these:
'''
() => {
  return <ButtonGrid variant="cta" label="CTA" />;
}
'''
We need to make sure the target Story is simpler if we're going to link like this